libpam/include/pam_inline.h

```diff
@@ -66,6 +66,60 @@ pam_str_skip_icase_prefix_len(const char *str, const char *prefix, size_t prefix
 #define pam_str_skip_icase_prefix(str_, prefix_)	\
 	pam_str_skip_icase_prefix_len((str_), (prefix_), sizeof(prefix_) - 1 + PAM_MUST_BE_ARRAY(prefix_))
 
+
+/*
+ * Macros to securly erase memory
+ */
+
+#ifdef HAVE_MEMSET_EXPLICIT
+# define _pam_override_n(x, n)           \
```

Comment:
Rather than override I'd use _pam_cleanup() or even pam_overwrite() without the underscore - given this is internal macro we do not need to particularly care about conflicts.